### PR TITLE
Place clusters show up in panel

### PR DESCRIPTION
**Pull request by mutaphysis**

### CODE REVIEW

File: src/components/Map/ClusterIcon.js [hunk 1]
```diff
@@ -4,56 +4,93 @@ import L from 'leaflet';
...
+import { interpolateWheelchairAccessibility } from '../../lib/colors';
 import isSamePlace from './isSamePlace';
 import getIconNameForProperties from './getIconNameForProperties';
-import * as categoryIcons from '../icons/categories';
+import * as markers from '../icons/markers';
+import Icon, { StyledIconContainer } from '../Icon';
 
 export default class ClusterIcon extends L.Icon {
   constructor(options: typeof L.Icon.options) {
+    // increased tap region for icons, rendered size might differ
```
Comment (opyh) [inline src/components/Map/ClusterIcon.js]: -1: Have you tested that this does not accidentally cover PoIs below clusters and makes them unreachable via tap?

File: src/components/Map/ClusterIcon.js [hunk 2]
```diff
@@ -4,56 +4,93 @@ import L from 'leaflet';
...
+      popupAnchor: new L.Point(size * 0.5, size * 0.5),
+      tooltipAnchor: new L.Point(size * 0.5, size * 0.5 + 25),
     };
 
     super(Object.assign(defaults, options));
   }
 
   createIcon() {
-    const div = document.createElement('div');
+    const elem = document.createElement('a');
```
Comment (opyh) [inline src/components/Map/ClusterIcon.js]: -1: A11y tests dislike `<a>` without href – how about `<button>`?

File: src/components/Map/Map.js [hunk 3]
```diff
@@ -168,15 +173,30 @@ export default class Map extends React.Component<Props, State> {
...
 
-    const zoom = overrideZoom || fallbackZoom;
-
-    const placeOrEquipment = state.placeOrEquipment;
-
+    let zoom = overrideZoom || fallbackZoom;
     let center = [0, 0];
     let bounds = null;
     let zoomedToFeatureId = state.zoomedToFeatureId;
+    let centerDefined = false;
```
Comment (opyh) [inline src/components/Map/Map.js]: I have to admit I don't fully understand the logic behind this flag… could you add a comment what it does? Maybe renaming is even enough :)

File: src/components/NodeToolbar/NodeToolbar.js [hunk 4]
```diff
@@ -357,7 +357,7 @@ class NodeToolbar extends React.Component<Props, State> {
         <ErrorBoundary>
           <FocusTrap
             component="div"
-            // We need to set clickOutsideDeactivates here as we want clicks on e.g. the map markers to not be pervented.
+            // We need to set clickOutsideDeactivates here as we want clicks on e.g. the map markers to not be prevented.
```
Comment (opyh) [inline src/components/NodeToolbar/NodeToolbar.js]: …perverted!

File: src/lib/colors.js [hunk 5]
```diff
@@ -85,39 +85,134 @@ const interpolateYesLimited = interpolateLab(
...
-export function interpolateWheelchairAccessibilityColors(propertiesArray: NodeProperties[]) {
+function calculateWheelchairAccessibility(propertiesArray: NodeProperties[]) {
   if (!propertiesArray || propertiesArray.length === 0) {
-    return colors.markers.background.unknown;
+    return { definedCount: 0 };
   }
-  const accessibilityValues = propertiesArray.map(isWheelchairAccessible);
+
+  // take a 'random' selection when there are too many places
+  const filterMod = Math.floor(propertiesArray.length / 30);
```
Comment (opyh) [inline src/lib/colors.js]: -1: lodash has a nicer-looking `sample` method :)

I think it's a good idea, but out of curiosity, did this noticeably change speed?

File: src/components/Map/Map.css [hunk 6]
```diff
@@ -229,75 +229,51 @@ body.is-touch-device .leaflet-control-zoom {
   fill: white;
 }
 
-.ac-marker-current {
```
Comment (opyh) [inline src/components/Map/Map.css]: +35: Finally!

File: src/components/Map/ClusterIcon.js (hunk 2 above)
Comment (mutaphysis) [reply src/components/Map/ClusterIcon.js]: Just aligned with Icon.js

File: src/components/Map/ClusterIcon.js (hunk 1 above)
Comment (mutaphysis) [reply src/components/Map/ClusterIcon.js]: The same effect was in place before, just in Map.css. Clustering radius takes the increased click area into account, and we never had complaints so far.

File: src/components/Map/Map.js (hunk 3 above)
Comment (mutaphysis) [reply src/components/Map/Map.js]: Will do!

File: src/lib/colors.js (hunk 5 above)
Comment (mutaphysis) [reply src/lib/colors.js]: I didn’t measure really. But I was running with the react inspector plugin, this is pretty slow already. I can add some data on this.

**Pull request merged**